dailyblink/blinks.py

```diff
@@ -126,6 +128,9 @@ def set_m4a_meta_data(
     genre=None,
 ):
     tags = MP4(filename).tags
+	if tags is None:
```

Comment:
this simply skips tag creation. see https://github.com/ptrstn/dailyblink/pull/11